client/components/gsuite/gsuite-new-user-list/new-user.tsx

```diff
@@ -52,7 +52,7 @@ const GSuiteNewUser: FunctionComponent< GSuiteNewUserProps > = ( {
 	selectedDomainName,
 	showAllErrors = false,
 	showTrashButton = true,
-} ) => {
+}: GSuiteNewUserProps ) => {
```

Comment:
As above RE the return type.
```suggestion
}: GSuiteNewUserProps ): ReactElement => {
```